tools/ci_build/github/apple/c/onnxruntime-c.config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "onnxruntime-c",
+    "summary": "ONNX Runtime C/C++ Pod",
+    "description": "A pod for the ONNX Runtime C/C++ library."
```

Comment:
@manashgoswami FYI, the description/summary for the packages is in these config.json files. There are also some in tools/ci_build/github/apple/objectivec.